webapp/components/test-run.html

```diff
@@ -59,6 +59,11 @@
   </template>
 
   <script>
+    const versionPatterns = {
```

Comment:
From https://eslint.org/docs/developer-guide/code-conventions

> Variables that act as constants (values that won’t be changed) should be formatted using all uppercase letters with words separated by a single underscore.

Reply:
It's surprising that the ESLint project would recommend that without offering a linting rule to enforce it.